docs/use-mainnet/bridges-of-linea/how-to-bridge-eth.md

```diff
@@ -0,0 +1,188 @@
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/pending-transaction-l1-to-l2.png")
+      .default
+  }></img>
+
+7. You’re done! Under Recent Transactions, it should say "Bridging complete" and your ETH should be on Linea Mainnet. _Reminder: It takes around 20 minutes for the bridging process to complete._
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/confirmed-transaction-l1-to-l2.png")
+      .default
+  }></img>
+
+8. You can see your Linea wallet balance in MetaMask once you switch to the Linea network. You can also check it on [lineascan](https://lineascan.build/address/0x331FB12C080F5b34F0E8812D44114D17398A016d). Just replace the wallet address in the URL with your wallet address and it should reflect your wallet's balance!
+
+## Manual Claiming
+
+1. Go to our token bridge [here](https://bridge.linea.build/).
+
+2. Connect your wallet in top right corner of the page.
+
+   <img
+     src={
+       require("@site/static/img/docs/use-mainnet/bridges-of-linea/linea-bridge-connect.png")
+         .default
+     }></img>
+
+3. Verify that the Bridge is set to `Ethereum → Linea Mainnet`, and that **manual claiming** is enabled.
+
+4. Enter the amount of ETH you want to bridge over to Linea Mainnet, and select the **Start Bridging** button.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/manual-claiming.png")
+      .default
+  }></img>
+
+5. MetaMask will pop up asking you to confirm the transaction.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/metamask-confirmation.png")
+      .default
+  }></img>
+
+6. You will see a “Claim Funds” button appear under the Recent Transactions section. Select the "Claim Funds" button to continue the bridging process.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/claim-funds.png")
+      .default
+  }></img>
+
+7. MetaMask will prompt you to switch networks and to also confirm the transaction from the previous step.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/claim-message-metamask.png")
+      .default
+  }></img>
+
+8. You’re done! Under Recent Transactions, it should say "Bridging complete" and your ETH should be on Linea Mainnet. _Reminder: It takes around 20 minutes for the bridging process to complete._
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/Confirmation-manual-claim.png")
+      .default
+  }></img>
+
+9. You can see your Linea wallet balance in MetaMask once you switch to the Linea network. You can also check it on [linea-scan](https://lineascan.build/address/0x331FB12C080F5b34F0E8812D44114D17398A016d). Just replace the wallet address in the URL with your wallet address and it should reflect your wallet's balance!
```

Comment:
_account_ balance rather than wallet balance? 

Not sure if this would require more rephrasing, like: "You can see the balance of your account on Linea on MetaMask" (but that feels clumsy)